server/gitrest/packages/gitrest-base/package.json

```diff
@@ -52,15 +52,15 @@
 	},
 	"dependencies": {
 		"@fluidframework/common-utils": "^1.1.1",
-		"@fluidframework/gitresources": "^4.0.0",
-		"@fluidframework/protocol-base": "^4.0.0",
+		"@fluidframework/gitresources": "^5.0.0-250395",
+		"@fluidframework/protocol-base": "^5.0.0-250395",
```

Comment:
You shopuldn't use range dependencies on prerelease versions. Use exact versions instead.

Reply:
Oh, my bad! Thanks for the call out